[submodule] Update fbgemm pinned version

gchalump opened the pull request:
Summary:
Update fbgemm pinned version in PyTroch.
Related update in fbgemm: D74434751

Included changes:
Update fbgemm external dependencies directory in setup.py
Add DISABLE_FBGEMM_AUTOVEC flag to disable fbgemm's autovec

Test Plan: PyTorch OSS CI

Differential Revision: D75073516

Skylion007 commented on cmake/Dependencies.cmake:
```diff
@@ -681,6 +681,10 @@ if(USE_FBGEMM)
+
     # TODO: Remove next two lines after fbgemm pin is updated
 
     # For more details see https://github.com/pytorch/pytorch/issues/150846
```
@gchalump can these two lines be removed?

gchalump committed:
[submodule] Update fbgemm pinned version (#153950)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/153950

Update fbgemm pinned version in PyTroch.
Related update in fbgemm: D74434751

Included changes:
Update fbgemm external dependencies directory in setup.py
Add DISABLE_FBGEMM_AUTOVEC flag to disable fbgemm's autovec

Test Plan: PyTorch OSS CI

Differential Revision: D75073516

Skylion007 commented:
We really need to enable LTO at some point to shrink these binaries down to avoid this static linking situation

gchalump commented:
@pytorchbot merge
